ros_cross_compile/platform.py

```diff
@@ -61,7 +61,10 @@ class Platform:
     """
 
     def __init__(
-        self, arch: str, os_name: str, ros_distro: str, override_base_image: Optional[str] = None,
+        self,
+        arch: str, os_name: str, ros_distro: str,
+        override_base_image: Optional[str] = None,
+        testing_version: bool = False,
```

Comment:
Is there a reason why some are on separate lines and not all/none?

Reply:
Even though it is a boolean statement, I think `testing_version` would be more human-readable as an enum.
ex:
`Platform('x86_64', 'debian', 'melodic', VersionType.TESTING)`